src/molecules/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SearchBar from './SearchBar';
+
+describe('Search Bar works properly', () => {
+  it('contains the search icon', () => {
+    render(<SearchBar />);
+    const searchBarContainer = screen
+      .getByRole('textbox', { placeholder: /Search.../i })
+      .closest('div');
+    const searchIcon = screen.getByRole('img');
+    expect(searchBarContainer).toContainElement(searchIcon);
+  });
+
+  it('lets you write in the search bar', () => {
+    render(<SearchBar />);
+    const searchBar = screen.getByRole('textbox', { placeholder: /Search.../i });
+    userEvent.type(searchBar, 'charizard');
```

Comment:
Before you trigger the event, you can check that the input has a different (or empty) value. This way you are making sure that the value actually changes when this event gets triggered. Also, I didn't know about `userEvent`, I usually use `fireEvent` from `@tesint-library/react`. I read that userEvent uses fireEvent behind the scenes, it is good to know it. Thanks.